homeassistant/components/backup/manager.py

```diff
@@ -435,6 +435,9 @@ async def _async_upload_backup(
                 # no point in continuing
                 raise BackupManagerError(str(result)) from result
             if isinstance(result, BackupAgentError):
+                LOGGER.error(
+                    "Error uploading to %s %s", agent_ids[idx], result, exc_info=result
```

Comment:
We should only include the stack trace for unknown errors.
```suggestion
                    "Error uploading to %s: %s", agent_ids[idx], result,
```